login/daint-mc/21.09

```diff
@@ -31,15 +31,16 @@ if { [module-info mode remove] || [module-info mode switch] } {
  foreach module ${toolsmclist} {
    module unload ${module}
  }
-# unload cdt and cdt-cuda
- module unload cdt cdt-cuda
 }
 
 # load dependency and update MODULEPATH
 module load craype-broadwell
 module use "${mcpath}"
 module use "${toolsmcpath}"
 
+# load additional modules
+#module load xalt
```

Comment:
Same question here about `xalt` as above @victorusu

Reply:
yes, we should

Reply:
I am not sure. Let's talk about this before changing the default behaviour..